cdb2jdbc/src/main/java/com/bloomberg/comdb2/jdbc/Comdb2Handle.java

```diff
@@ -758,6 +758,14 @@ private boolean isClientOnlySetCommand(String sql) {
         return false;
     }
 
+    private boolean isBeginQuery(String sql) {
+        if (sql.equalsIgnoreCase("begin") ||
+            sql.ReplaceAll("\\s+"," ").equalsIgnoreCase("start transaction")) {
```

Comment:
This is wrong. It does not capture `START TRANSACTION AS OF`.

Reply:
I missed that, pushed a fix. I wonder how ```if (lowerSql.equals("begin"))``` worked in case of AS OF? Probably, the scenario was never tested with jdbc driver?

Reply:
Yes. That is a bug too. Nice catch.